### DISCUSSION TITLE
embrace-the-impedance-mismatch/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#23] [Blog comments]
# embrace-the-impedance-mismatch/

https://blogs.taiga.nl/martijn/posts/embrace-the-impedance-mismatch/

**Comment by disqus-comments-migrator** [comment]
<em>Erwin Berends commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at december 05 2008, 06:37</em>

---

<p>I totally agree!</p>
